debug/src/debug.js

```diff
@@ -233,22 +233,23 @@ export function initDebug() {
 				}
 			}
 
-			let values = vnode.props;
-			if (vnode.type._forwarded) {
+			// If vnode is not present we're mounting
+			let values = vnode ? vnode.props : internal.props;
```

Comment:
FYI: At this point `internal` will point to the `props` object from the previous render, not the new one.